### DISCUSSION TITLE
How to build docs?

### DISCUSSION THREAD

**Discussion by bwkam** [alpinejs/alpine#4051] [1. Help]
title

**Comment by ekwoka** [comment]
You can't. The site is in a different repo.

**Reply by bwkam** [reply]
It's private? Because navigating the markdown is pain for me, and the website isn't loading at all for me.

edit: the website seems to work now for some reason, so closing this.

**Reply by ekwoka** [reply]
The actual site is a separate private Laravel/Livewire repo that just pulls the markdown from this repo.

It contains some things that are paid products as well as the public documentation.
